### PR TITLE
Upgraded TaxiFarePrediction solution to v0.11

### PR DESCRIPTION
**Pull request by prathyusha12345**
## Summary

Upgraded TaxiFarePrediction solution to v0.11 with the below changes.
1.Removed unnecessary namespaces.
2.Removed a code in code snippet which is removed in Ml.Net v0.11. 
3.Reirdered the arguments order in API calls.

### CODE REVIEW

File: machine-learning/tutorials/TaxiFarePrediction/Program.cs [hunk 1]
```diff
@@ -31,22 +25,7 @@ static void Main(string[] args)
...
-                                new TextLoader.Column("RateCode", DataKind.Text, 1),
-                                new TextLoader.Column("PassengerCount", DataKind.R4, 2),
-                                new TextLoader.Column("TripTime", DataKind.R4, 3),
-                                new TextLoader.Column("TripDistance", DataKind.R4, 4),
-                                new TextLoader.Column("PaymentType", DataKind.Text, 5),
-                                new TextLoader.Column("FareAmount", DataKind.R4, 6)
-                            }
-            }
-            );
+            //This code has been removed in v0.11.
```
Comment (prathyusha12345) [inline machine-learning/tutorials/TaxiFarePrediction/Program.cs]: Please check this as the code is been removed according ML.Net 0.11

Comment (JRAlexander) [reply machine-learning/tutorials/TaxiFarePrediction/Program.cs]: Can you remove snippet4 please? Otherwise looks good and I'll merge

Comment (prathyusha12345) [reply machine-learning/tutorials/TaxiFarePrediction/Program.cs]: @JRAlexander Ok I will remove <snippet4> and push changes.

**Comment by prathyusha12345** [comment]
@JRAlexander I did the requested change. Can you please merge once you review.

**Comment by JRAlexander** [comment]
Excellent, @prathyusha12345! Thanks for your help!

**Pull request merged**